src/api_proxy/service_control/request_info.h

```diff
@@ -166,7 +182,12 @@ struct QuotaRequestInfo : public OperationInfo {
 
 // Stores the information extracted from the quota response.
 struct QuotaResponseInfo {
-  ScResponseErrorType error_type = ScResponseErrorType::ERROR_TYPE_UNSPECIFIED;
+  ScResponseErrorType type;
```

Comment:
why do we still need "type"?  is it covered by "error"?

Reply:
Yes, I think so. @nareddyt 

This PR is very big already. We can refactor in the next PR.

Reply:
Seems you wrapped `ScResponseErrorType` inside `ScResponseError`, so you should be able to remove it. This is the same thing you did for CheckResponseInfo